packages/components/popconfirm/src/popconfirm.vue

```diff
@@ -71,6 +74,12 @@ const hidePopper = () => {
   tooltipRef.value?.onClose?.()
 }
 
+const style = computed(() => {
+  return {
+    width: addUnit(props.width),
+  }
+})
+
```

Comment:
## Suggest

I think this is a very nice feature in my opinion!

additionally I use box-sizing: border-box; I think the attribute should be added.

## Why

![image](https://user-images.githubusercontent.com/27342882/181906530-9b9ed764-c1de-433f-9138-355a2217b625.png)

you can specify the exact size by adding a property to `.el-popover.el-popper.`

![image](https://user-images.githubusercontent.com/27342882/181906581-d530a3fd-37ae-4745-b26d-c053f9a82e72.png)

Reply:
Thank you very much for your suggestion, I think it is very good, I will add it. 😆